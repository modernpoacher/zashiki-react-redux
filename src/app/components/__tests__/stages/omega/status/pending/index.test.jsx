import React from 'react'
import renderer from 'react-test-renderer'

import Pending from '#app/components/stages/omega/status/pending'

describe('#app/components/stages/omega/status/pending', () => {
  describe('Always', () => {
    it('renders', () => {
      const component = (
        <Pending />
      )

      expect(renderer.create(component).toJSON())
        .toMatchSnapshot()
    })
  })
})
