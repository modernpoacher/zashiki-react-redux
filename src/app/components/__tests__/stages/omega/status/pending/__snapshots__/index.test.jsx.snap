// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#app/components/stages/omega/status/pending Always renders 1`] = `
<div
  className="omega pending"
/>
`;
